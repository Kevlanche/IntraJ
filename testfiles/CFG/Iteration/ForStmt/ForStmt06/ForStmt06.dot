digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[18]"; "Opt[19]"; "Exit[20]"}
{ rank = same; "List[15]"; "Block[20]"}
{ rank = same; "List[21]"}
{ rank = same; "VarDeclStmt[22]"; "VarDeclStmt[30]"; "ForStmt[38]"; "VarDeclStmt[70]"}
{ rank = same; "Modifiers[23]"; "PrimitiveTypeAccess[25]"; "List[26]"; "Modifiers[31]"; "PrimitiveTypeAccess[33]"; "List[34]"; "List[39]"; "Opt[53]"; "List[57]"; "Block[64]"; "Modifiers[71]"; "PrimitiveTypeAccess[73]"; "List[74]"}
{ rank = same; "List[24]"; "VariableDeclarator[27]"; "List[32]"; "VariableDeclarator[35]"; "VarDeclStmt[40]"; "LTExpr[54]"; "ExprStmt[58]"; "ExprStmt[61]"; "List[65]"; "List[72]"; "VariableDeclarator[75]"}
{ rank = same; "List[28]"; "Opt[29]"; "List[36]"; "Opt[37]"; "Modifiers[41]"; "PrimitiveTypeAccess[43]"; "List[44]"; "VarAccess[55]"; "IntegerLiteral[56]"; "PostIncExpr[59]"; "PostDecExpr[62]"; "ExprStmt[66]"; "List[76]"; "Opt[77]"}
{ rank = same; "List[42]"; "VariableDeclarator[45]"; "VariableDeclarator[49]"; "VarAccess[60]"; "VarAccess[63]"; "AssignSimpleExpr[67]"}
{ rank = same; "List[46]"; "Opt[47]"; "List[50]"; "Opt[51]"; "VarAccess[68]"; "IntegerLiteral[69]"}
{ rank = same; "IntegerLiteral[48]"; "IntegerLiteral[52]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < ForStmt06 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[19]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[20]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[21]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[22]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[23]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[25]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[27]" [label="VariableDeclarator\n < int a >", fillcolor=white   style=filled  ]
"List[28]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[29]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[30]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[31]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[33]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[35]" [label="VariableDeclarator\n < int b >", fillcolor=white   style=filled  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[37]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ForStmt[38]" [label="ForStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[39]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[40]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[41]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[42]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[43]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[44]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[45]" [label="VariableDeclarator\n < int i >", fillcolor=white   style=filled  ]
"List[46]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[47]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[48]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"VariableDeclarator[49]" [label="VariableDeclarator\n < int j >", fillcolor=white   style=filled  ]
"List[50]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[51]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[52]" [label="IntegerLiteral\n < 1 >", fillcolor=white   style=filled  ]
"Opt[53]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"LTExpr[54]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[55]" [label="VarAccess\n < i >", fillcolor=white   style=filled  ]
"IntegerLiteral[56]" [label="IntegerLiteral\n < 10 >", fillcolor=white   style=filled  ]
"List[57]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[58]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PostIncExpr[59]" [label="PostIncExpr\n < i >", fillcolor=white   style=filled  ]
"VarAccess[60]" [label="VarAccess\n < i >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[61]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PostDecExpr[62]" [label="PostDecExpr\n < j >", fillcolor=white   style=filled  ]
"VarAccess[63]" [label="VarAccess\n < j >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[64]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[65]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[66]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[67]" [label="AssignSimpleExpr\n < a >", fillcolor=white   style=filled  ]
"VarAccess[68]" [label="VarAccess\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[69]" [label="IntegerLiteral\n < 8 >", fillcolor=white   style=filled  ]
"VarDeclStmt[70]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[71]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[72]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[73]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[74]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[75]" [label="VariableDeclarator\n < int c >", fillcolor=white   style=filled  ]
"List[76]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[77]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[20]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[18]" [style=dashed, color=gray]
"Modifiers[23]" -> "List[24]" [style=dashed, color=gray]
"VarDeclStmt[22]" -> "Modifiers[23]" [style=dashed, color=gray]
"VarDeclStmt[22]" -> "PrimitiveTypeAccess[25]" [style=dashed, color=gray]
"VariableDeclarator[27]" -> "List[28]" [style=dashed, color=gray]
"VariableDeclarator[27]" -> "Opt[29]" [style=dashed, color=gray]
"List[26]" -> "VariableDeclarator[27]" [style=dashed, color=gray]
"VarDeclStmt[22]" -> "List[26]" [style=dashed, color=gray]
"List[21]" -> "VarDeclStmt[22]" [style=dashed, color=gray]
"Modifiers[31]" -> "List[32]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "Modifiers[31]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "PrimitiveTypeAccess[33]" [style=dashed, color=gray]
"VariableDeclarator[35]" -> "List[36]" [style=dashed, color=gray]
"VariableDeclarator[35]" -> "Opt[37]" [style=dashed, color=gray]
"List[34]" -> "VariableDeclarator[35]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "List[34]" [style=dashed, color=gray]
"List[21]" -> "VarDeclStmt[30]" [style=dashed, color=gray]
"Modifiers[41]" -> "List[42]" [style=dashed, color=gray]
"VarDeclStmt[40]" -> "Modifiers[41]" [style=dashed, color=gray]
"VarDeclStmt[40]" -> "PrimitiveTypeAccess[43]" [style=dashed, color=gray]
"VariableDeclarator[45]" -> "List[46]" [style=dashed, color=gray]
"Opt[47]" -> "IntegerLiteral[48]" [style=dashed, color=gray]
"VariableDeclarator[45]" -> "Opt[47]" [style=dashed, color=gray]
"List[44]" -> "VariableDeclarator[45]" [style=dashed, color=gray]
"VariableDeclarator[49]" -> "List[50]" [style=dashed, color=gray]
"Opt[51]" -> "IntegerLiteral[52]" [style=dashed, color=gray]
"VariableDeclarator[49]" -> "Opt[51]" [style=dashed, color=gray]
"List[44]" -> "VariableDeclarator[49]" [style=dashed, color=gray]
"VarDeclStmt[40]" -> "List[44]" [style=dashed, color=gray]
"List[39]" -> "VarDeclStmt[40]" [style=dashed, color=gray]
"ForStmt[38]" -> "List[39]" [style=dashed, color=gray]
"LTExpr[54]" -> "VarAccess[55]" [style=dashed, color=gray]
"LTExpr[54]" -> "IntegerLiteral[56]" [style=dashed, color=gray]
"Opt[53]" -> "LTExpr[54]" [style=dashed, color=gray]
"ForStmt[38]" -> "Opt[53]" [style=dashed, color=gray]
"PostIncExpr[59]" -> "VarAccess[60]" [style=dashed, color=gray]
"ExprStmt[58]" -> "PostIncExpr[59]" [style=dashed, color=gray]
"List[57]" -> "ExprStmt[58]" [style=dashed, color=gray]
"PostDecExpr[62]" -> "VarAccess[63]" [style=dashed, color=gray]
"ExprStmt[61]" -> "PostDecExpr[62]" [style=dashed, color=gray]
"List[57]" -> "ExprStmt[61]" [style=dashed, color=gray]
"ForStmt[38]" -> "List[57]" [style=dashed, color=gray]
"AssignSimpleExpr[67]" -> "VarAccess[68]" [style=dashed, color=gray]
"AssignSimpleExpr[67]" -> "IntegerLiteral[69]" [style=dashed, color=gray]
"ExprStmt[66]" -> "AssignSimpleExpr[67]" [style=dashed, color=gray]
"List[65]" -> "ExprStmt[66]" [style=dashed, color=gray]
"Block[64]" -> "List[65]" [style=dashed, color=gray]
"ForStmt[38]" -> "Block[64]" [style=dashed, color=gray]
"List[21]" -> "ForStmt[38]" [style=dashed, color=gray]
"Modifiers[71]" -> "List[72]" [style=dashed, color=gray]
"VarDeclStmt[70]" -> "Modifiers[71]" [style=dashed, color=gray]
"VarDeclStmt[70]" -> "PrimitiveTypeAccess[73]" [style=dashed, color=gray]
"VariableDeclarator[75]" -> "List[76]" [style=dashed, color=gray]
"VariableDeclarator[75]" -> "Opt[77]" [style=dashed, color=gray]
"List[74]" -> "VariableDeclarator[75]" [style=dashed, color=gray]
"VarDeclStmt[70]" -> "List[74]" [style=dashed, color=gray]
"List[21]" -> "VarDeclStmt[70]" [style=dashed, color=gray]
"Block[20]" -> "List[21]" [style=dashed, color=gray]
"Opt[19]" -> "Block[20]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[19]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[20]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"PostDecExpr[62]" -> "VarAccess[55]" [color=blue, constraint=false, xlabel=" "] 
"PostDecExpr[62]" -> "PostIncExpr[59]" [color=red, constraint=false]
"PostIncExpr[59]" -> "PostDecExpr[62]" [color=blue, constraint=false, xlabel=" "] 
"PostIncExpr[59]" -> "AssignSimpleExpr[67]" [color=red, constraint=false]
"AssignSimpleExpr[67]" -> "PostIncExpr[59]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[67]" -> "IntegerLiteral[69]" [color=red, constraint=false]
"IntegerLiteral[69]" -> "AssignSimpleExpr[67]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[69]" -> "LTExpr[54]" [color=red, constraint=false]
"Exit[20]" -> "VariableDeclarator[75]" [color=red, constraint=false]
"VariableDeclarator[75]" -> "Exit[20]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[75]" -> "LTExpr[54]" [color=red, constraint=false]
"LTExpr[54]" -> "IntegerLiteral[69]" [color=blue, constraint=false, xlabel="T "] 
"LTExpr[54]" -> "VariableDeclarator[75]" [color=blue, constraint=false, xlabel="F "] 
"LTExpr[54]" -> "IntegerLiteral[56]" [color=red, constraint=false]
"IntegerLiteral[56]" -> "LTExpr[54]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[56]" -> "VarAccess[55]" [color=red, constraint=false]
"VarAccess[55]" -> "IntegerLiteral[56]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[55]" -> "VariableDeclarator[49]" [color=red, constraint=false]
"VarAccess[55]" -> "PostDecExpr[62]" [color=red, constraint=false]
"VariableDeclarator[49]" -> "VarAccess[55]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[49]" -> "IntegerLiteral[52]" [color=red, constraint=false]
"IntegerLiteral[52]" -> "VariableDeclarator[49]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[52]" -> "VariableDeclarator[45]" [color=red, constraint=false]
"VariableDeclarator[45]" -> "IntegerLiteral[52]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[45]" -> "IntegerLiteral[48]" [color=red, constraint=false]
"IntegerLiteral[48]" -> "VariableDeclarator[45]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[48]" -> "VariableDeclarator[35]" [color=red, constraint=false]
"VariableDeclarator[35]" -> "IntegerLiteral[48]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[35]" -> "VariableDeclarator[27]" [color=red, constraint=false]
"VariableDeclarator[27]" -> "VariableDeclarator[35]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[27]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "VariableDeclarator[27]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[18]" -> "Opt[19]" -> "Exit[20]" [style=invis]
"VarDeclStmt[22]" -> "VarDeclStmt[30]" -> "ForStmt[38]" -> "VarDeclStmt[70]" [style=invis]
"Modifiers[23]" -> "PrimitiveTypeAccess[25]" -> "List[26]" [style=invis]
"List[28]" -> "Opt[29]" [style=invis]
"Modifiers[31]" -> "PrimitiveTypeAccess[33]" -> "List[34]" [style=invis]
"List[36]" -> "Opt[37]" [style=invis]
"List[39]" -> "Opt[53]" -> "List[57]" -> "Block[64]" [style=invis]
"Modifiers[41]" -> "PrimitiveTypeAccess[43]" -> "List[44]" [style=invis]
"VariableDeclarator[45]" -> "VariableDeclarator[49]" [style=invis]
"List[46]" -> "Opt[47]" [style=invis]
"List[50]" -> "Opt[51]" [style=invis]
"VarAccess[55]" -> "IntegerLiteral[56]" [style=invis]
"ExprStmt[58]" -> "ExprStmt[61]" [style=invis]
"VarAccess[68]" -> "IntegerLiteral[69]" [style=invis]
"Modifiers[71]" -> "PrimitiveTypeAccess[73]" -> "List[74]" [style=invis]
"List[76]" -> "Opt[77]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
79", fillcolor=1]
Edges [label="Total Nr. Edges
110", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
16", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
16", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}