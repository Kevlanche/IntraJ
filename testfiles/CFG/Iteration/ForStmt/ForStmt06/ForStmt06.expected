NodeName: Entry[13] Successors: [VariableDeclarator[27]]
NodeName: VariableDeclarator[27] Successors: [VariableDeclarator[35]]
NodeName: VariableDeclarator[35] Successors: [IntegerLiteral[48]]
NodeName: IntegerLiteral[48] Successors: [VariableDeclarator[45]]
NodeName: VariableDeclarator[45] Successors: [IntegerLiteral[52]]
NodeName: IntegerLiteral[52] Successors: [VariableDeclarator[49]]
NodeName: VariableDeclarator[49] Successors: [VarAccess[55]]
NodeName: VarAccess[55] Successors: [IntegerLiteral[56]]
NodeName: IntegerLiteral[56] Successors: [LTExpr[54]]
NodeName: LTExpr[54] Successors: [IntegerLiteral[69], VariableDeclarator[75]]
NodeName: IntegerLiteral[69] Successors: [AssignSimpleExpr[67]]
NodeName: AssignSimpleExpr[67] Successors: [PostIncExpr[59]]
NodeName: PostIncExpr[59] Successors: [PostDecExpr[62]]
NodeName: PostDecExpr[62] Successors: [VarAccess[55]]
NodeName: VariableDeclarator[75] Successors: [Exit[20]]
NodeName: Exit[20] Successors: []

NodeName: Exit[20] Predecessor: [VariableDeclarator[75]]
NodeName: VariableDeclarator[75] Predecessor: [LTExpr[54]]
NodeName: LTExpr[54] Predecessor: [IntegerLiteral[56]]
NodeName: IntegerLiteral[56] Predecessor: [VarAccess[55]]
NodeName: VarAccess[55] Predecessor: [VariableDeclarator[49], PostDecExpr[62]]
NodeName: VariableDeclarator[49] Predecessor: [IntegerLiteral[52]]
NodeName: IntegerLiteral[52] Predecessor: [VariableDeclarator[45]]
NodeName: VariableDeclarator[45] Predecessor: [IntegerLiteral[48]]
NodeName: IntegerLiteral[48] Predecessor: [VariableDeclarator[35]]
NodeName: VariableDeclarator[35] Predecessor: [VariableDeclarator[27]]
NodeName: VariableDeclarator[27] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: PostDecExpr[62] Predecessor: [PostIncExpr[59]]
NodeName: PostIncExpr[59] Predecessor: [AssignSimpleExpr[67]]
NodeName: AssignSimpleExpr[67] Predecessor: [IntegerLiteral[69]]
NodeName: IntegerLiteral[69] Predecessor: [LTExpr[54]]
